src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -668,6 +674,24 @@ protected DynamoModel(IStartConfiguration config)
             extensionManager.MessageLogged += LogMessage;
             var extensions = config.Extensions ?? LoadExtensions();
 
+            //when dynamo is ready, alert the loaded extensions
```

Comment:
A minor point of annoyance. Please add a space between your `//` and the text. This is most definitely the style employed in this code base.